bundles/org.openhab.binding.hdpowerview/src/main/resources/OH-INF/i18n/hdpowerview.properties

```diff
@@ -0,0 +1,39 @@
+# binding
+
+binding.hdpowerview.name = Hunter Douglas PowerView Binding
+binding.hdpowerview.description = The Hunter Douglas PowerView binding provides access to the Hunter Douglas line of PowerView shades.
+
+# thing types
+
+thing-type.hdpowerview.hub.label = PowerView Hub
+thing-type.hdpowerview.hub.description = Hunter Douglas (Luxaflex) PowerView Hub
+thing-type.hdpowerview.shade.label = PowerView Shade
+thing-type.hdpowerview.shade.description = Hunter Douglas (Luxaflex) PowerView Shade
+
+# thing types config
+
+thing-type.config.hdpowerview.hub.hardRefresh.label = Hard Position Refresh Interval
+thing-type.config.hdpowerview.hub.hardRefresh.description = The number of minutes between hard refreshes of positions from the PowerView Hub (or 0 to disable)
+thing-type.config.hdpowerview.hub.hardRefreshBatteryLevel.label = Hard Battery Level Refresh Interval
+thing-type.config.hdpowerview.hub.hardRefreshBatteryLevel.description = The number of hours between hard refreshes of battery levels from the PowerView Hub (or 0 to disable, default is weekly)
+thing-type.config.hdpowerview.hub.host.label = Host
+thing-type.config.hdpowerview.hub.host.description = The Host address of the PowerView Hub
+thing-type.config.hdpowerview.hub.refresh.label = Refresh Interval
+thing-type.config.hdpowerview.hub.refresh.description = The number of milliseconds between fetches of the PowerView Hub shade state
+thing-type.config.hdpowerview.shade.id.label = ID
+thing-type.config.hdpowerview.shade.id.description = The numeric ID of the PowerView Shade in the Hub
+
+# channel types
+
+channel-type.hdpowerview.battery-voltage.label = Battery Voltage
+channel-type.hdpowerview.battery-voltage.description = Battery voltage reported by the shade
+channel-type.hdpowerview.scene-activate.description = Activates the scene ''{0}''
+channel-type.hdpowerview.scene-group-activate.description = Activates the scene group ''{0}''
+channel-type.hdpowerview.shade-position.label = Position
+channel-type.hdpowerview.shade-position.description = The vertical position of the shade
```

Comment:
Two entries are missing for label and description of the channel `secondary`.
Keys should be probably `thing-type.hdpowerview.shade.channel.secondary.label` and `thing-type.hdpowerview.shade.channel.secondary.description`.
Please try the result before submitting.

Reply:
Added and verified to be correct. Thanks for this, couldn't find these keys in the documentation. Seems like I missed this commit in the tool for auto-generating: https://github.com/openhab/openhab-core/pull/2544/commits/414c85c9ddcc3311afef86092d70196ab82e32cb